### ISSUE TITLE
Create "Feedback, Receiving" slideshow

### ISSUE THREAD

**Issue by vegetabill** [Techtonica/curriculum#1152]
We would like to give more guidance around asking for feedback and dealing with it when you receive it.

### Topics to Cover
- What are ways to ask for feedback?
  - e.g. asking "Any feedback for me?" is too broad and doesn't work
- What to do when you receive difficult feedback on you or your code?

Suggest some small group activities to do during the presentation so participants can get practice.

Total estimate time would be 30-45min including group discussion / activities.

### Bonus
- Videos of talks like this they can view for more info

### Format
Please clone this [Google Slides Template](https://docs.google.com/presentation/d/1xIEehDgCj7AyePY-ERZ8gQJ7jPRvFDXyaIDdDBFY3UI/edit#slide=id.p) for the slides.

See also: #1151 - Giving

**Comment by vegetabill** [comment]
https://docs.google.com/presentation/d/1kaTYV_PoKpMukLSAev4MmjfFR02nTUCfjwD6rRptbk0/edit#slide=id.p

**Comment by alodahl** [comment]
@vegetabill Does this need to be added to an outline or the Table of Contents?

**Comment by vegetabill** [comment]
Yes let's add this!! I couldn't find the other slideshow either. Reopening

**Comment by alodahl** [comment]
Is there an outline you wanted to add to here, or do we need to create a new one @vegetabill ?  We like to have things in both slideshow and text form if we can.

**Comment by vegetabill** [comment]
@alodahl I think it was meant to be a new lesson so I updated this issue and #1151 to be accurate and added them to the first H1-2021 milestone.

**Comment by alodahl** [comment]
Same with this one... are you personally working on resolving this issue?

**Comment by vegetabill** [comment]
Oops no. Unassigning! All my energy will likely be going toward helping get Eventonica ship shape before it begins.

**Comment by Lakhwinderr** [comment]
I created a lesson.md file. Please review and merge #2104.

**Comment by daaimah123** [comment]
@Lakhwinderr Merged and labeled "hacktoberfest-accepted"
